__tests__/index.spec.ts

```diff
@@ -52,3 +52,54 @@ it('should accept custom config', async () => {
   expect(news[0]).has.property('url').which.is.a('string')
   expect(news[0]).has.property('date').which.is.a('date')
 })
+
+it('should filter by two dates', async () => {
+  const news = await getNews({
+    region: 'na',
+    category: 'topics',
+    count: 100,
+    after: new Date(1658217599 * 1000), // 2022-07-19 07:59:59 UTC+0
+    before: new Date('2022-07-20'),
+  })
+  expect(news).is.an('array')
+  expect(news[0]).has.property('title').which.is.a('string')
+  expect(news[0]).has.property('epoch').which.is.a('number')
+  expect(news[0]).has.property('url').which.is.a('string')
+  expect(news[0]).has.property('date').which.is.a('date')
+  expect(news.length).is.equal(2)
+  expect(news[0].date).is.lessThan(new Date('2022-07-20'))
+  expect(news[news.length - 1].date).is.greaterThan(new Date(1658217599 * 1000))
+}).timeout(0)
+
+it('should filter by one date', async () => {
+  const lodestone = new LodestoneNews({
+    region: 'na',
+    count: 100,
+  })
+  const news = await lodestone.getNews({
+    category: 'topics',
+    after: new Date(1658217599 * 1000), // 2022-07-19 07:59:59 UTC+0
+  })
```

Comment:
- Why not just call `getNews` as above.
```suggestion
  const news = getNews({
    category: 'topics',
    region: 'na',
    count: 100,
    after: new Date(1658217599 * 1000), // 2022-07-19 07:59:59 UTC+0
  })
```